code/style.css

```diff
@@ -1,58 +1,176 @@
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover; 
+  }
+
+.header {
+  top: 0;
+  z-index: 100;
+  width: 100%;
+  height: 70px;
+  text-align: center;
+  background-color: rgb(39, 5, 4);
+}
+
+.header-text {
+  display: inline-block;
+  font-size: 20px;
+  padding: 5px;
+  margin-right: 50px;
+  text-transform: uppercase;
+  font-family: 'Lacquer', sans-serif;
+  color: rgb(199, 184, 184);
 }
 
 p {
-  margin: 1px;
+  margin: 3px;
   font-weight: 300;
-  background-color: #fff;
+  color: beige;
+  background-color: rgba(68, 10, 8, 0.514);
+  font-family: 'Lacquer', sans-serif;
 }
 
 .cards {
-  margin: 50px auto;
-  width: 90%;
-  height: 600px;
-  background: #ccc;
   display: flex;
+  flex-direction: row;
   justify-content: space-around;
   align-items: center;
+  margin: 20px auto;
+  width: 90%;
+  height: auto;
+  padding-top: 40px;
+  cursor: pointer;
 }
 
 .killer-deck {
   width: 250px;
   height: 400px;
-  background: #000;
+  background: rgba(0, 0, 0, 0.137);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 5px;
   border-radius: 16px;
+  margin-bottom: 10px;
+  background-image: url(./assets/hands.jpg);
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover; 
+  cursor: pointer;
+}
+
+.killer-deck:hover {
+  border: 1px solid;
+  padding: 5px;
+  box-shadow: 5px 5px #a3a2a2;
 }
 
 .weapon-deck {
   width: 250px;
   height: 400px;
-  background: rgb(10, 53, 10);
+  background: rgba(0, 0, 0, 0.137);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 5px;
   border-radius: 16px;
+  background-image: url(./assets/smoke.jpg);
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover; 
+  cursor: pointer;
+}
+
+.weapon-deck:hover {
+  border: 1px solid;
+  padding: 5px;
+  box-shadow: 5px 5px #a3a2a2;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
-  background: midnightblue;
-
+  background: rgba(0, 0, 0, 0.137);
   text-align: center;
-  padding-top: 30px;
+  padding-top: 5px;
   border-radius: 16px;
+  background-image: url(./assets/steps.jpg);
+  background-size: 100%;
+  cursor: pointer;
+}
+
+.room-deck:hover {
+  border: 1px solid;
+  padding: 5px;
+  box-shadow: 5px 5px #a3a2a2;
+}
+ 
+
+h1 {
+  padding: 8px 20px;
+  border: none;
+  border-radius: 8px;
+  display: block;
+  position: relative;
+  margin: 0 auto;
+  color: rgb(255, 255, 255);
+  text-shadow: 5px 5px rgb(58, 7, 7);
+  font-size: 30px;
+  font-family: 'Lacquer', sans-serif;
+}
+
+#crimebutton {
+  color: rgb(199, 184, 184);
+  font-size: 25px;
 }
 
 button {
-  padding: 8px 16px;
-  background: crimson;
+  padding: 8px 20px;
+  background: rgb(43, 7, 3);
   border: none;
-  font-size: 20px;
   border-radius: 8px;
   display: block;
   margin: 0 auto;
+  cursor: pointer;
+  font-family: 'Lacquer', sans-serif;
```

Comment:
You seem to use this font a lot.  Is there another way to make it used on the entire page?